tests/AutoReview/ProjectFixerConfigurationTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\AutoReview;
+
+use PhpCsFixer\Console\ConfigurationResolver;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @coversNothing
+ * @group auto-review
+ */
+final class ProjectFixerConfigurationTest extends TestCase
+{
+    public function testCreate()
+    {
+        $setLocation = __DIR__.'/../../.php_cs.dist';
+
+        $this->assertFileExists($setLocation, 'Failed to find ".php_cs.dist".');
+        $this->assertTrue(is_readable($setLocation), sprintf('Config file "%s" is not readable.', $setLocation));
+
+        /** @var \PhpCsFixer\Config $config */
+        $config = require $setLocation;
```

Comment:
prior 2 assertions are pointless, as if file is not there/ is not readable, without having those assertions this test will fail as well